test/screenshot/spec/mdc-textfield/classes/disabled-character-counter.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
Can we use single screenshot test for all disabled text fields? We can adjust the size of viewport since it doesn't fit in `test-viewport--mobile`.

We want to reduce the number of screenshot counts. Also makes the migration easier.

Reply:
Could we defer this as a separate effort? We can work out a plan to thin down screenshot counts as a whole across components in a unified way.

Reply:
I think any new screenshots from now on should be limited and clubbed into one big screenshot when possible. Similar to how we test text field internally.

Otherwise, this'll get only piled up and makes our tests even slower. We do not want to refactor or unify our existing screenshots since we're migrating testing infra internally.